amundsen_application/static/js/components/common/SearchBar/InlineSearchResults/tests/index.spec.tsx

```diff
@@ -0,0 +1,432 @@
+
+jest.mock('config/config-utils', () => ({
+  getDatabaseDisplayName: jest.fn(),
+  getDatabaseIconClass: jest.fn(),
+  indexUsersEnabled: jest.fn(),
+}));
+import { getDatabaseDisplayName, getDatabaseIconClass, indexUsersEnabled } from 'config/config-utils';
+
+import globalState from 'fixtures/globalState';
+import { allResourcesExample } from 'fixtures/search/inlineResults';
+
+describe('InlineSearchResults', () => {
+  const setup = (propOverrides?: Partial<InlineSearchResultsProps>) => {
+    const props: InlineSearchResultsProps = {
+      isLoading: false,
+      tables: allResourcesExample.tables as SearchResults<TableResource>,
+      users: allResourcesExample.users as SearchResults<UserResource>,
+      className: 'testClass',
+      onItemSelect: jest.fn(),
+      searchTerm: 'test search',
+      ...propOverrides
+    };
+    const wrapper = shallow<InlineSearchResults>(<InlineSearchResults {...props} />);
+    return { props, wrapper };
+  };
+
+  describe('getTitleForResource', () => {
+    let wrapper;
+    beforeAll(() => {
+      wrapper = setup().wrapper;
+    });
+    it('returns the correct value for ResourceType.table', () => {
+      const output = wrapper.instance().getTitleForResource(ResourceType.table);
+      expect(output).toEqual(CONSTANTS.DATASETS);
+    });
+    it('returns the correct value for ResourceType.user', () => {
+      const output = wrapper.instance().getTitleForResource(ResourceType.user);
+      expect(output).toEqual(CONSTANTS.PEOPLE);
+    });
+    it('returns empty string as the default', () => {
+      const output = wrapper.instance().getTitleForResource('unsupported');
+      expect(output).toEqual('');
+    });
+  });
+
+  describe('getTotalResultsForResource', () => {
+    let props;
+    let wrapper;
+    beforeAll(() => {
+      const setupResult = setup();
+      props = setupResult.props;
+      wrapper = setupResult.wrapper;
+    });
+    it('returns the correct value for ResourceType.table', () => {
+      const output = wrapper.instance().getTotalResultsForResource(ResourceType.table);
+      expect(output).toEqual(props.tables.total_results);
+    });
+    it('returns the correct value for ResourceType.user', () => {
+      const output = wrapper.instance().getTotalResultsForResource(ResourceType.user);
+      expect(output).toEqual(props.users.total_results);
+    });
+    it('returns 0 as the default', () => {
+      const output = wrapper.instance().getTotalResultsForResource('unsupported');
+      expect(output).toEqual(0);
+    });
+  });
+
+  describe('getResultsForResource', () => {
+    let props;
+    let wrapper;
+    beforeAll(() => {
+      const setupResult = setup();
+      props = setupResult.props;
+      wrapper = setupResult.wrapper;
+    });
+    it('returns the correct sliced results for ResourceType.table', () => {
+      const output = wrapper.instance().getResultsForResource(ResourceType.table);
+      expect(output).toEqual(props.tables.results.slice(0, 2));
```

Comment:
I'll make it a constant so it's easier for us to change. 

As far as it being a config, I think it should be in the long run but opted against it because our search service is limited to 10 results per page anyway. I think it would be nice to make this a config when our results per page are also configurable.